client/src/backend/components/project/hero/Builder/BlockHeaderDetail.js

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import Utility from "global/components/utility";
+import PropTypes from "prop-types";
+
+const BlockHeaderDetail = ({ titleId, title, description }) => {
+  return (
+    <>
+      <div className="hero-builder-block__header-details">
+        <h3 id={titleId} className="hero-builder-block__title">
+          {title}
+        </h3>
+        {!!description && (
+          <p className="hero-builder-block__description">{description}</p>
+        )}
+      </div>
+      <div className="hero-builder-block__button">
+        <span className="hero-builder-block__button-label">Edit</span>
+        <Utility.IconComposer icon="annotate32" size={26} />
+      </div>
+    </>
+  );
+};
+
+BlockHeaderDetail.propTypes = {
+  title: PropTypes.string.isRequired,
+  titleId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  description: PropTypes.string,
+  hasDescription: PropTypes.bool
+};
+
+BlockHeaderDetail.propTypes = {
+  titleId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string
+};
```

Comment:
@dananjohnson Before you merge this I noticed, the duplicate `propTypes` here. Totally something I did. 🙃 The second set, sans `hasDescription` is correct.

Reply:
Good eye! Will fix then merge.